Update mobile-windows-universal-dotnet-authenticate-app-with-token.md

niteshrestha opened the pull request:
LoginAsync method should have been LoginAsync(provider, "{url_scheme_of_your_app}"); instead of LoginAsync(provider);

azurecla commented:
Hi __@flanakin__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (micflan). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

niteshrestha committed:
Update mobile-windows-universal-dotnet-authenticate-app-with-token.md

LoginAsync method should have been LoginAsync(provider, "{url_scheme_of_your_app}"); instead of LoginAsync(provider);
